jerry-core/ecma/operations/ecma-jobqueue.h

```diff
@@ -26,6 +26,28 @@
  */
 
 /**
+ * Description of the PromiseReactionJob
+ */
+typedef struct
+{
+  ecma_value_t reaction; /**< the PromiseReaction */
+  ecma_value_t argument; /**< argument for the reaction */
+} ecma_job_promise_reaction_t;
```

Comment:
It is only used in ecma-jobquue.c, and i will put the definition into the source file.